apds9960/apds9960_nano_33_ble.go

```diff
@@ -0,0 +1,28 @@
+// +build nano_33_ble
+
+package apds9960
+
+import (
+	"machine"
+	"time"
+
+	"tinygo.org/x/drivers"
+)
+
+// New creates a new APDS-9960 connection. The I2C bus must already be
+// configured.
+//
+// This function only creates the Device object, it does not touch the device.
+func New(bus drivers.I2C) Device {
+	// turn on internal power pin (machine.P0_22) and I2C1 pullups power pin (machine.P1_00)
```

Comment:
Comment does not match the code?

Reply:
Which part? "it does not touch the devce"? In a sense it is, because this only turns on the power supply.

Reply:
No, I mean the pin numbers

Reply:
Ok, I guess I mixed up the internal I2C pins.